R/plot_fixed_effects_para.R

```diff
@@ -0,0 +1,75 @@
+#' Plot the annual parameter estimates from the fit model
+#'
+#'
+#' @param data List created by the sdmTMB:: fit function
+#' @template dir 
+#'
+#' @import sdmTMB
+#'
+#' @author Chantel Wetzel
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#'  plot_fixed_effects_para(
+#'        data = fit, 
+#'        dir = file.path(sppdir, survey, "index", obs))
+#' }
+#'
+#'
+plot_fixed_effects_para <- function(data, dir) {
+
+  est <- as.list(data$sd_report, "Estimate", report = FALSE)
```

Comment:
This is all fine as is, but just in case you weren't aware, `tidy(fit, conf.int = TRUE)` can help with this type of thing. Also `tidy(fit, "ran_pars", conf.int = TRUE)` and `tidy(fit, "ran_vals", conf.int = TRUE)`.  See `?sdmTMB:::tidy.sdmTMB`. There's a `model` argument for delta models.

Reply:
Fixed in a5372b5